### PR TITLE
[OSCD] Powershell Disable Windows Defender AV

### PR DESCRIPTION
**Pull request by invrep-de**
This identifies attempts to disable Windows Defender from powershell that is commonly leveraged by malicious threat actors, such as EvilCorp/TA505, Lazarus, and others.

### CODE REVIEW

File: rules/windows/process_creation/win_powershell_disable_windef_av.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+title: Powershell Used To Disable Windows Defender AV Security Monitoring
+id: a7ee1722-c3c5-aeff-3212-c777e4733217
+status: experimental
+description: Detects attackers attempting to disable Windows Defender using Powershell
+author: ok @securonix invrep-de, oscd.community
```
Comment (yugoslavskiy) [inline rules/windows/process_creation/win_powershell_disable_windef_av.yml:5]: hello @invrep-de! thank you for your contribution! could you please add quotes in the beginning and in the end of the `author` field value?

Comment (invrep-de) [reply rules/windows/process_creation/win_powershell_disable_windef_av.yml]: Thanks, fixed

Comment (yugoslavskiy) [reply rules/windows/process_creation/win_powershell_disable_windef_av.yml]: Thank you, @invrep-de!

**Review by yugoslavskiy — APPROVED** [review]
@thomaspatzke @Neo23x0 guys, I believe that this PR could be merged (:
please let us know what do you think about it if there is anything you would like to change.

**Pull request closed without merging**